scripts/ovmd

```diff
@@ -109,17 +112,9 @@ kill_old_podman() {
 make_fs() {
 	BLKNODE="/dev/$BLKNAME"
 	UUID="8a3219d0-4002-4cd9-8cb1-f3ffe52451f1"
+	findfs_bin="/usr/bin/findfs"
 
-	lsblk_bin="/usr/bin/lsblk"
-	test -f ${lsblk_bin} && {
-		# Find lsblk
-		echo -n ""
-	} || {
-		# Not find lsblk
-		exit 100
-	}
-
-	${lsblk_bin} -o UUID ${BLKNODE} | grep "${UUID}"
+	${findfs_bin} UUID=${UUID} >/dev/null 2>&1 
```

Comment:
Do we need to determine if the block returned by stdout is the block we expected?

Reply:
After discussion, no additional judgments will be made here for the time being, because the issue is too borderline.